workspaces/liferay-partner-workspace/client-extensions/liferay-partner-site-initializer-code/site-initializer/object-fields/12-account.json

```diff
@@ -16,7 +16,7 @@
 		},
 		{
 			"DBType": "String",
-			"businessType": "Picklist",
+			"businessType": "MultiselectPicklist",
```

Comment:
@JoMaAlves like we talked about it will probably be easier for rollout to create a new field and populate the data before getting rid of (or in this case updating) the old field